ext/standard/tests/php_ini_scanned_files.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+string php_ini_scanned_files ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
+    die('SKIP php version too low.');
+}
+?>
+--FILE--
+<?php
+var_dump(is_string(php_ini_scanned_files()));
+?>
+--EXPECTF--
+bool(%s)
```

Comment:
That only checks that is_string() returns a bool. Not very meaningful with regard to php_ini_scanned_files(). It would be best to check what php_ini_scanned_files() actually returns for a faked environment (not sure if that's possible, though).